tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -125,10 +125,18 @@ def cmd(self):
     def post_checks(self):
         return []
 
+    @abc.abstractmethod
+    def pre_cmd(self):
+        return []
+
+    @abc.abstractmethod
+    def hrr_post_checks(self, named_group):
+        return []
```

Comment:
Optional suggestion. With it , only keep function in MBedTLS is enough
```suggestion
    
    def hrr_post_checks(self, named_group):
        return []
```

Reply:
I am not sure, seems pylint will complain it without abstrat.